api/src/services/eml-service.ts

```diff
@@ -49,36 +52,254 @@ type SurveyObjectWithAttachments = SurveyObject & {
   report_attachments?: GetSurveyReportAttachmentsData;
 };
 
-type Cache = {
-  projectData?: IGetProject;
-  surveyData?: SurveyObjectWithAttachments[];
-  projectAttachmentData?: GetProjectAttachmentsData;
-  projectReportAttachmentData?: GetProjectReportAttachmentsData;
-  codes?: IAllCodeSets;
+type ProjectMetadataSource = {
+  projectData: IGetProject;
+  projectAttachmentsData: GetProjectAttachmentsData;
+  projectReportAttachmentsData: GetProjectReportAttachmentsData;
```

Comment:
I can confirm with Charlie during our meeting this afternoon at 2, but I suspect we will be removing attachment related information from the EML file, now that we have that information being sent separately. I forgot we even had it in here originally tbh.

Reply:
As per dicusssion with Charlie, we're removing attachment/report information in both project and survey EML. Acceptance criteria have been changed accordingly